tests/oeedger8r/host/testbasic.cpp

```diff
@@ -32,8 +32,8 @@ void test_basic_edl_ecalls(oe_enclave_t* enclave)
             255,
             19,
             20,
-            21) == OE_OK);
-
+            21,
+            22) == OE_OK);
```

Comment:
This should be 21 with your change; not 22 as the master has since been changed (my new PR #1147 ).